content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md

```diff
@@ -102,7 +102,7 @@ Get insights into the performance or the adoption of a given team's scope by usi
    - Errors by service
    - Page views by service
 
-{{< img src="real_user_monitoring/guide/define-applications-services-components-rum/rum-guide-rum-applications-overview-page.jpg" alt="Search query for actions grouped by user name on Shopist's Cart page" style="width:90%;">}}
+{{< img src="real_user_monitoring/guide/define-applications-services-components-rum/rum-guide-rum-applications-overview-page-1.png" alt="Search query for actions grouped by user name on Shopist's Cart page" style="width:90%;">}}
```

Comment:
could we have a red box around where the service filter is here? turns out it's not that obvious when a value is selected

Reply:
I struggled to find it while staring at the screenshot 😆 updated!